src/command_modules/azure-cli-hdinsight/azure/cli/command_modules/hdinsight/custom.py

```diff
@@ -202,6 +207,8 @@ def create_cluster(cmd, client, cluster_name, resource_group_name, cluster_type,
                 key=storage_account_key,
                 container=storage_default_container,
                 file_system=storage_default_filesystem,
+                resource_id=storage_account if not is_wasb else None,
```

Comment:
Suggest `None if is_wasb else storage_account` to get rid of the `not`

Reply:
Good suggestion! I am still on my way of learning Python:)